docs/sql_reference/src/asciidoc/_chapters/sql_functions_and_expressions.adoc

```diff
@@ -11290,6 +11295,99 @@ ID
 --- 1 row(s) selected.
 ```
 
+<<<
```

Comment:
Should there be a SYS_GUID example somewhere also?

Reply:
Hi, Dave, we already have a SYS_GUID example

Reply:
OK, thanks.